### PR TITLE
Leaf-3224 service chief reloaded after remove

### PR DESCRIPTION
**Pull request by jampaul3**
This makes the functionality the same for service chiefs as it is for user groups.

### CODE REVIEW

File: LEAF_Request_Portal/admin/Group.php [hunk 1]
```diff
@@ -144,10 +144,10 @@ public function removeSyncGroup(int $groupID): array
     {
         $sql_vars = array(':groupID' => $groupID);
         $this->dataActionLogger->logAction(\DataActions::DELETE, \LoggableTypes::PORTAL_GROUP, [
-            new \LogItem("users", "groupID", $groupID, $this->getGroupName($groupID))
+            new \LogItem("groups", "groupID", $groupID, $this->getGroupName($groupID))
         ]);
 
-        $result = $this->db->prepared_query('DELETE FROM users WHERE groupID=:groupID', $sql_vars);
+        $result = $this->db->prepared_query('DELETE FROM groups WHERE groupID=:groupID', $sql_vars);
```
Comment (raphaelfretter) [inline LEAF_Request_Portal/admin/Group.php:150]: This should probably remove the users from said group as well. I'm not sure a cascade delete is implemented in the groups table.

File: LEAF_Request_Portal/sources/Service.php [hunk 2]
```diff
@@ -236,30 +248,42 @@ public function removeMember($groupID, $member)
...
 
-            $this->db->prepared_query('DELETE FROM service_chiefs WHERE userID=:userID AND serviceID=:groupID', $sql_vars);
-
             // check if this service is also an ELT
             $sql_vars = array(':groupID' => $groupID);
-            $res = $this->db->prepared_query('SELECT * FROM services
-   												WHERE serviceID=:groupID', $sql_vars);
+            $sql = 'SELECT groupID
+                    FROM services
+   					WHERE serviceID = :groupID
```
Comment (raphaelfretter) [inline LEAF_Request_Portal/sources/Service.php:271]: Spacing

File: LEAF_Request_Portal/sources/System.php [hunk 3]
```diff
@@ -947,8 +947,9 @@ public function syncSystem(\Group $org_group, \Service $org_service, \OrgChart\G
 
         // update services and service chiefs
         $services = $nexus_group->listGroupsByTag('service');
+        error_log(print_r($services, true));
```
Comment (raphaelfretter) [inline LEAF_Request_Portal/sources/System.php:950]: Should this be left in?

**Review by raphaelfretter — CHANGES_REQUESTED** [review]

File: LEAF_Request_Portal/admin/Group.php (hunk 1 above)
Comment (jampaul3) [reply LEAF_Request_Portal/admin/Group.php]: I just removed this as it isn't getting used any longer.

File: LEAF_Request_Portal/sources/System.php [hunk 4]
```diff
@@ -1050,7 +1055,7 @@ public function syncSystem(\Group $org_group, \Service $org_service, \OrgChart\G
             }
 
             foreach ($employees as $employee) {
-                if ($employee['userName'] != '') {
+                if (isset($employee['userName']) && !empty($employee['userName'])) {
```
Comment (Pelentan) [inline LEAF_Request_Portal/sources/System.php:1058]: empty() does isset().  Here and below.

**Review by Pelentan — CHANGES_REQUESTED** [review]

**Review by Pelentan — APPROVED** [review]

**Review by aerinkayne — APPROVED** [review]

**Review by pete-nerantzinis — APPROVED** [review]

**Review by raphaelfretter — APPROVED** [review]

**Pull request closed without merging**